tools/oozie/oozie_simple_test/README.md

```diff
@@ -0,0 +1,27 @@
+# Oozie Simple Tester
+
+Use these files for a simple oozie test case that runs a simple oozie shell action.
+
+In order to run the test, simple update coordinator.properties with:
+ - The yarn task tracker url
+ - The HDFS name node url
+ - An HDFS directory to store the files
+ - A start and end time set to today
+
+Then, create your test HDFS directory, and create 2 subdirectories underneath - lib and output.
+
+Finally, put the following files into that HDFS directory.
+ - coordinator.xml  
+ - logoozieid.sh  
+ - test.sh  
+ - workflow.xml
+
```

Comment:
Mention `kinit`'ing as a user first and creating a kerberos identity for one (e.g. `ubuntu`?) There's a pointer to doing that in our Wiki under [`Add Kerberos principals by using kadmin.local on bootstrap node`](https://github.com/bloomberg/chef-bach/wiki/Creating-a-Kerberized-cluster).